.github/workflows/build-test.yml

```diff
@@ -298,11 +297,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions-rs/toolchain@v1
+      - uses: dtolnay/rust-toolchain@nightly
         with:
-          toolchain: nightly
           components: clippy
-          override: true
+        run: rustup override set nightly
```

Comment:
is this necessary given you use `@nightly`?

Reply:
This is what the behavior of the old workflow was (`override: true`), didn't check if it's necessary. I believe it essentially just overrides the `toolchain.toml` file, though I'm not sure we have one of those. I can try removing it?

Reply:
I see